### ISSUE TITLE
Coercion from Vectorz to NDarray does not work

### ISSUE THREAD

**Issue by Gerrrr** [mikera/core.matrix#239]
``` Clojure
user=> m1
#<NDArray [[22.0 17.0] [17.0 31.0]]>
user=> (mp/coerce-param m1 (m/matrix :vectorz m1))

VectorzException 0D get not supported on matrix!  mikera.matrixx.AMatrix.get (AMatrix.java:104)
```

It works for other implementations, e.g.:

``` Clojure
user=> (mp/coerce-param m1 (m/matrix :clatrix m1))
#<NDArray [[22.0 17.0] [17.0 31.0]]>
user=> (mp/coerce-param m1 (m/matrix :persistent-vector m1))
#<NDArray [[22.0 17.0] [17.0 31.0]]>
```

**Comment by mikera** [comment]
I know why this is failing. Will figure out a fix shortly.

**Comment by mikera** [comment]
Fixed in this commit I think:
https://github.com/mikera/core.matrix/commit/c8cc190ba7c2909c619e41a391e36574d733b637

**Comment by Gerrrr** [comment]
Works for me, thanks! Can I ask you to make a minor release? Travis CI is failing Incanter because of it. Alternatively I can just disable test till next core.matrix release.

**Comment by mikera** [comment]
OK just released `0.36.1` with this fix

**Comment by Gerrrr** [comment]
Thank you!
